registry/txt.go

```diff
@@ -273,6 +289,13 @@ func (im *TXTRegistry) ApplyChanges(ctx context.Context, changes *plan.Changes)
 		UpdateOld: endpoint.FilterEndpointsByOwnerID(im.ownerID, changes.UpdateOld),
 		Delete:    endpoint.FilterEndpointsByOwnerID(im.ownerID, changes.Delete),
 	}
+
+	if im.isMigrationEnabled {
```

Comment:
lines not covered with unit tests

![Screenshot 2025-06-23 at 06 54 49](https://github.com/user-attachments/assets/8aa37c79-6301-44e8-a206-c20cef06f570)